docker-compose.yml

```diff
@@ -227,11 +236,96 @@ services:
       - --set=bundles.cvat.polling.min_delay_seconds=5
       - --set=bundles.cvat.polling.max_delay_seconds=15
 
+  clickhouse:
```

Comment:
Need to have a well-defined pattern to name services. Now some of them have "cvat_" prefix, and some of them don't.  I don't see the logic.

Reply:
add `cvat_` prefix